spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/MultiMethodRabbitListenerEndpoint.java

```diff
@@ -47,7 +52,36 @@ protected HandlerAdapter configureListenerAdapter(MessagingMessageListenerAdapte
 			invocableHandlerMethods.add(getMessageHandlerMethodFactory()
 					.createInvocableHandlerMethod(getBean(), method));
```

Comment:
Good catch; in fact we don't even need the field; we can `setBean()` from the ctor and use `getBean()` everywhere else.

Not sure why I would have added the field to the subclass.